var/spack/repos/builtin/packages/py-python-mapnik/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPythonMapnik(PythonPackage):
```

Comment:
Any idea if this package corresponds to https://pypi.org/project/mapnik/? If so, I'm inclined to name this `py-mapnik`. It looks like @flippmoke is the maintainer of the PyPI package, and has contributed to https://github.com/mapnik/python-mapnik/, so he can probably answer this question.

Reply:
Yeah, I noticed that there was some confusion regarding the name of the package.

Reply:
I honestly don't maintain the python-mapnik bindings anymore, not sure what is going on here though as I just got @ -ed in.

Reply:
Hey @flippmoke, sorry for the ping. We just wanted to make sure that https://pypi.org/project/mapnik/ and https://github.com/mapnik/python-mapnik/ are indeed the same project.

Reply:
> Any idea if this package corresponds to https://pypi.org/project/mapnik/? If so, I'm inclined to name this py-mapnik

The homepage for the PyPI site leads to the Mapnik homepage (https://mapnik.org/) rather than the Mapnik python bindings Github page so my assessment is that they are different (also the PyPI site latest artifacts are from 2015). So I think this name makes sense.